src/components/spatial/Spatial.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-
+import PubSub from 'pubsub-js';
 import {
   ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM, BitmapLayer, BaseTileLayer,
 } from 'deck.gl';
+import { MicroscopyViewerLayer } from '@hubmap/vitessce-image-viewer';
 import { load } from '@loaders.gl/core';
+import { SLIDERS_CHANGE, COLORS_CHANGE, CHANNEL_TOGGLE } from '../../events';
```

Comment:
... trying to remember why we didn't already need events here?

Reply:
I don't think the component itself listens for anything.  Now that I'm typing this, I guess It should move into the subscriber, then.